### ISSUE TITLE
Tab focus make optional

### ISSUE THREAD

**Issue by timeshift92** [carbon-design-system/carbon-components-svelte#572]
```
 afterUpdate(() => {
    if (didMount && selected && ref) {
      ref.focus();
    }
  });
  ```
  the current moment when a change in the tab component takes the focus on itself, there are cases where your input changes from the parent, then the component is updated, then the focus interferes

**Comment by metonym** [comment]
I agree that – in practice – programmatically updating the current index should not steal focus from another element.

**Comment by metonym** [comment]
Fixed in [v0.64.3](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.64.3).

The selected tab should no longer steal the focus if it was programmatically updated. See the demo in https://github.com/carbon-design-system/carbon-components-svelte/pull/1320
